src/main/resources/site/content-types/article/article.xml

```diff
@@ -180,5 +180,18 @@
                 </input>
             </items>
         </field-set>
+
+        <field-set>
+            <label>Kontakter</label>
+            <items>
+                <input name="contacts" type="CustomSelector">
+                    <occurrences minimum="1" maximum="0" />
```

Comment:
this shouldn't be forced, it should be possible to have an article without contacts. The part should then be hidden as well.